.concourse/suse-buildpacks-ci/buildpack-version-bump/tasks/build.sh

```diff
@@ -24,8 +24,9 @@ echo "${REGISTRY_PASS}" | docker login "${REGISTRY_NAME}" --username "${REGISTRY
 # Extract the fissile binary.
 tar xvf s3.fissile-linux/fissile-*.tgz --directory "/usr/local/bin/"
 
-# Pull the stemcell image.
-stemcell_version="$(cat s3.stemcell-version/"${STEMCELL_VERSIONED_FILE##*/}")"
+# Pull the stemcell image referenced in the KUBECF_VALUES
+sle_version="$(cat s3.stemcell-version/"${STEMCELL_VERSIONED_FILE##*/}" | cut -d- -f1)"
+stemcell_version=${sle_version}-$(perl -0ne '/stemcell:\n\s+version: (\d+\.\d+)/g&&print "$1"' "kubecf/${KUBECF_VALUES}")
```

Comment:
This is complicated enough that I'd like to have an intermediate variable here.

It appears that the image (`splatform/base-ci:kubecf`) has `yq` installed (the Python3 version); perhaps we can get the version string with that? I can't find the provenance of the docker image, though, which is a bit disconcerting.

Reply:
`yq` sounds like a great idea, but I did not quickly find a way to parse the `'$defaults'` entry because of the special characters.

Reply:
Cool, that works, many thanks.